src/Service/ValidationService.php

```diff
@@ -139,9 +139,9 @@ private function getEntityValidator(Entity $entity, int $level = 0): Validator
 
         $item->set($validator);
         // Tag for all Entity Validators.
-        $item->tag('entityValidator');
+        // $item->tag('entityValidator');
         // Tag for the Validators of this specific Entity.
-        $item->tag('entityValidator_'.$entity->getId()->toString());
+        // $item->tag('entityValidator_'.$entity->getId()->toString());
```

Comment:
Maybe remove the uncommented code if not being used anymore

Reply:
We might still want to cache these validators at some point.
@rjzondervan why did we do this change again that causes this way of caching to no longer work?

Reply:
Because otherwise caching fails altogether. If we can manage to use the cache adapter in a way that we do not break other caching we can reintroduce these tags again.